content/en/llm_observability/sdk.md

```diff
@@ -104,6 +104,12 @@ LLMObs.enable(
 : optional - _string_
 <br />The name of the service used for your application. If not provided, this defaults to the value of `DD_SERVICE`.
 
+### AWS Lambda setup
+
+Follow the setup instructions for the [Datadog-Python and Datadog-Extension][14] layers and LLM Observability will be automatically enabled.
```

Comment:
```suggestion
Follow the setup instructions for the [Datadog-Python and Datadog-Extension][14] layers to automatically enable LLM Observability.
```
Slightly confused on the wording here, does this mean that LLM Obs will be enabled and configured by following the layer setup instructions? Maybe we can improve the wording to be a bit more clear

Reply:
ah yeah, and with your next point too, I think I can probably combine these two points to something like:
```
Enable LLM Observability by specifying the required environment variables in [command line setup] and following the setup instructions for the [Datadog-Python and Datadog-Extension][14] AWS lambda layers. 

**Note**: Using the `Datadog-Python` and `Datadog-Extension` layers automatically turns on all LLM Observability integrations, and force flushes spans at the end of the lambda.
```

To kinda mirror the command line section. wdyt?